src/main/java/net/snowflake/ingest/streaming/internal/FlushService.java

```diff
@@ -358,28 +358,38 @@ void distributeFlushTasks() {
                         logger.logDebug(
                             "buildUploadWorkers stats={}", this.buildUploadWorkers.toString());
                         return buildAndUpload(filePath, blobData);
-                      } catch (IOException e) {
+                      } catch (IOException
```

Comment:
Should we have a generic `Exception` here or below? If something unexpected happens where does that get caught?